### ISSUE TITLE
Problem compiling samples.nim on Nim 0.19.0 

### ISSUE THREAD

**Issue by wizzardx** [Vladar4/libtcod_nim#10]
When running build_samples.sh, I first get this error:

```
samples.nim(7, 8) Error: cannot open file: libtcod
```
I need to work around that by editing samples.nim line 7, and changing this line:

```
import libtcod, os, parseutils, math
```
To this:
```
import ../libtcod, os, parseutils, math
```

My next problem after that is this error:

```
../libtcod_nim/private/console.nim(163, 14) Error: usage of 'isNil' is a user-defined error
```

Which I can work around by adding a sample.nims file with these contents:
```
--nilseqs:on
```

After which, the 'samples' binary compiles and runs fine.

**Comment by Vladar4** [comment]
1) Fixed in ed11a68f9cae49237466da5ff4efb0636e9571a4 (v0.98.1)

2) isNil checks are already removed in the v0.98.1, check if you are installing the latest version.

**Comment by wizzardx** [comment]
Thanks, these seem to be fixed for me now.
